mods/resize-text.user.js

```diff
@@ -0,0 +1,487 @@
+    const notifButtons = document.querySelectorAll('div.pills form[action="/settings/notifications/read"] button.btn');
+    const notifSizeMultiply = parseFloat(settings["optionNotifs"]) * 1.5;
+
+    // user notifs *loops*
+    userNotifications.forEach(userNotifsSelect => {
+        const userNotifsElem = userNotifsSelect.querySelectorAll('h2, div select, div div, input, textarea, markdown-toolbar, time, form.me-2 button[id="submit"]');
+        const userNotifsElemH1 = userNotifsSelect.querySelectorAll('h1, label[for="message_body"]');
+
+        userNotifsElem.forEach(userNotifResize => {
+            userNotifResize.style.setProperty('font-size', fontSizes.userNotifs);
+        })
+
+        userNotifsElemH1.forEach(userNotifResizeH1 => {
+            userNotifResizeH1.style.setProperty('font-size', `${notifSizeMultiply}px`);
+        })
+
+        notifButtons.forEach(notifButtonResize => {
+            const notifPurge = notifButtonResize.querySelectorAll('.btn.btn__secondary span');
+
+            notifButtonResize.style.setProperty('font-size', fontSizes.userNotifs);
+
+            notifPurge.forEach(notifPurgeResize => {
+                notifPurgeResize.style.setProperty('font-size', fontSizes.userNotifs);
+            })
+        })
+
+    })
+
+// === SORT BY === // 
+
+    // sort by *variables*
+    const sortBy = document.querySelectorAll('aside#options menu li a, aside#options menu i');
+
+    // sort by *loops*
+    sortBy.forEach(sortByElem => {
+        sortByElem.style.setProperty('font-size', fontSizes.sortBy);
+    })
+
+// === Footer === // 
+
+    //footer *variables*
+    const footerUseful = document.querySelectorAll('footer#footer');
+    const footerMultiply = parseFloat(settings["optionFooter"]) * 1.5;
+
+    //footer *loops*
+    footerUseful.forEach(footerSelect => {
+        const footerElems = footerSelect.querySelectorAll('section menu li a, section div a, div li a, i, select[data-action="kbin#changeLang"], #text');
+        const footerH1 = footerSelect.querySelectorAll('section h5');
+
+        footerElems.forEach(footerResize => {
+            footerResize.style.setProperty('font-size', fontSizes.footer);
+        });
+
+        footerH1.forEach(footerH1Resize => {
+            footerH1Resize.style.setProperty('font-size', `${footerMultiply}px`);
+        })
+    })
+
+// === ACTIVITY === //
+    const activity = document.querySelectorAll('div.section.users.users-columns');
+
+    // create posts *loops*
+    activity.forEach(activityElem => {
+        const activityElement = activityElem.querySelectorAll('ul, li, small, a, img');
+
+        activityElement.forEach(activityResize => {
+                activityResize.style.setProperty('font-size', fontSizes.activity);
+        })
+    });
+
+// === OPACITY FUNCTIONALITY === //
+    let opacity = settings.opacity;
+    const kesModalContent = document.querySelector('div.kes-settings-modal-content');
```

Comment:
I think this should be wrapped in a `try` statement so that if the modal doesn't exist, it doesn't spill `TypeError: kesModalContent is null` messages to the console. This happens if you are entering a page and not on the modal itself.